devito/dle/blocking_utils.py

```diff
@@ -399,6 +399,8 @@ def unfold(self):
 
 class BlockDimension(IncrDimension):
 
+    is_PerfKnob = True
```

Comment:
?

Reply:
types.Basic defines  `is_PerfKnob` and has a comment about that is